packaging/windows/split.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const chunkSize = 1024 * 1024 * 1024 // 1GiB chunk size
+
+func main() {
+	if len(os.Args) != 2 {
+		log.Fatal("Split takes only one argument (the file to split)")
+	}
+
+	if err := split(os.Args[1]); err != nil {
+		log.Fatal(err.Error())
+	}
+}
+
+func split(filePath string) error {
+	bundle, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer bundle.Close()
+	bundleName := filepath.Base(filePath)
+	for i := 0; ; i++ {
+		partFileName := fmt.Sprintf("%s.%d", bundleName, i)
+		partFile, err := os.Create(filepath.Join(filepath.Dir(filePath), partFileName))
+		if err != nil {
+			return err
+		}
+		defer partFile.Close()
+		n, err := io.CopyN(partFile, bundle, chunkSize)
+		fmt.Printf("Copied %d bytes from %s to %s\n", n, bundleName, partFileName)
+
+		if err != nil && !errors.Is(err, io.EOF) {
+			return err
+		}
```

Comment:
You need a `return nil` when you get `io.EOF` or this will be an infinite loop. I got this when running it:
```
...
Copied 0 bytes from crc_libvirt_4.5.4.crcbundle to crc_libvirt_4.5.4.crcbundle.1014
Copied 0 bytes from crc_libvirt_4.5.4.crcbundle to crc_libvirt_4.5.4.crcbundle.1015
Copied 0 bytes from crc_libvirt_4.5.4.crcbundle to crc_libvirt_4.5.4.crcbundle.1016
2021/04/09 13:08:23 open /home/teuf/redhat/crc/releases/crc_libvirt_4.5.4.crcbundle.1017: too many open files
```
(it kept looping, and only stopped because it hit the open file descriptor limit)

Reply:
Ouch, i hit this too, was to be fixed then forgot, thanks

Reply:
Changed, although now i feel like the nested if was better to read..

Reply:
If your "now" refers to the latest change you did, it's related to moving to `CopyN`, I don't think it's related to using a `func()` or not.

Reply:
its not about the anonymous function or use of `copyN`..